Makefile

```diff
@@ -59,6 +59,14 @@ up: config ## Start all services
 	@$(MAKE) showenv
 	@$(MAKE) showpass
 
+.PHONY: agup
```

Comment:
Can we parameterise `up` and `down` instead, along the lines of https://stackoverflow.com/a/2214793/1559300:

(e.g. `make up composition=docker-compose-ag.yml` and the default if not provided being `docker-compose.yml`)